examples/analog_hamiltonian_simulation/06_Analog_Hamiltonian_simulation_with_PennyLane.ipynb

```diff
@@ -22,7 +22,9 @@
    "cell_type": "code",
```

Comment:
<p>Shots are defined = 100 here, but the cost tracker mentions as 200. Can you check on that? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Yes, that is correct, because we run two tasks, each has 100 shots.